docs/users.rst

```diff
@@ -1015,6 +1021,9 @@ when verifying artifact was downloaded properly. At least one of them is
 required. If more than one is defined, multiple hashes will be computed
 and verified.
 
+If source-url is specified, the source-md5, source-sha1 or source-sha256
```

Comment:
also explicitly say, that one of those hashes HAS to be provided

Reply:
updated.